src/main/java/core/basesyntax/opationstrategy/OperationHandler.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.opationstrategy;
```

Comment:
Let’s  work with package names 
```suggestion
package core.basesyntax.strategy;
```
Please  use simple names